lib/src/main/java/com/orange/ods/compose/component/list/OdsListItem.kt

```diff
@@ -309,3 +324,88 @@ private interface OdsListItemDividerModifier : Modifier.Element {
 
     val startIndent: Dp?
 }
+
+@Composable
+@ExperimentalMaterialApi
+private fun PreviewOdsListItem(parameter: OdsListItemPreviewParameter) = Preview {
+    val iconType = parameter.second
+    val iconPainterResId = when (iconType) {
+        OdsListItemIconType.Icon -> R.drawable.ic_address_book
+        OdsListItemIconType.CircularImage,
+        OdsListItemIconType.SquareImage,
+        OdsListItemIconType.WideImage -> R.drawable.placeholder_small
+        null -> null
+    }
+    OdsListItem(
+        modifier = Modifier.let { modifier ->
+            iconType?.let { modifier.iconType(it) }.orElse { modifier }
+        },
+        text = "Text",
+        icon = iconPainterResId?.let { { OdsListItemIcon(painter = painterResource(it)) } },
+        secondaryText = parameter.first.first,
+        singleLineSecondaryText = parameter.first.second,
+        trailing = parameter.third
+    )
+}
+
+@Preview(name = "OdsListItem - Light")
+@Composable
+@ExperimentalMaterialApi
+private fun PreviewOdsListItemLight(@PreviewParameter(OdsListItemPreviewParameterProvider::class) parameter: OdsListItemPreviewParameter) {
+    PreviewOdsListItem(parameter)
+}
+
+@Preview(
+    name = "OdsListItem - Dark",
+    uiMode = Configuration.UI_MODE_NIGHT_YES,
+    showBackground = true
+)
+@Composable
+@ExperimentalMaterialApi
+private fun PreviewOdsListItemDark(@PreviewParameter(OdsListItemPreviewParameterProvider::class) parameter: OdsListItemPreviewParameter) {
+    PreviewOdsListItem(parameter)
+}
+
+typealias OdsListItemPreviewParameter = Triple<Pair<String?, Boolean>, OdsListItemIconType?, (@Composable () -> Unit)?>
```

Comment:
Following our discussion, we can replace the `Triple` by a data class to be more clear